core/signupcheck.php

```diff
@@ -15,18 +15,34 @@
     }
 
     else{
+   
+        $nametest=trim($_POST['Name']);
+        $enrolltest=trim($_POST['Enroll']);
+        $emailtest=trim($_POST['Email']);
```

Comment:
Why you are using two variables for same thing

Reply:
a single variable can do the work

Reply:
That is because, Name and Password in this case can actually contain spaces. And I dont want to lose the orignal value for eg- the name Manthan Surkar should stay the same and not ManthanSurkar

Reply:
@prateek76  is it something that I am missing here?